recipes/nodebook/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "nodebook" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "b0cbb9cee2deea33b3dc3d1c195b3b96a636dcdec18201cb99eb59f9c0d93a63" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/stitchfix/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - ipython
+    - jupyter
+    - click
+    - dill
+    - msgpack-python
+    - pandas
+    - pytest-runner
```

Comment:
Is `pytest-runner` actually needed as a runtime dependency, or is it just a testing dependency?

Reply:
It's listed in the `install-requires` packages in `setup.py`.  But yeah, I doubt it is necessary.  Will remove it and see if it still works on my machine.